src/static/js/modules/show-hide-fields.js

```diff
@@ -0,0 +1,38 @@
+/* ==========================================================================
+ * Show/Hide Additional Fields
+ *
+ * Shows additional fields on forms when additional information is needed
+ *
+ * ========================================================================== */
+
+'use strict';
+
+var $ = require( 'jquery' );
+
+function init() {
+
+  function toggleAdditionalFields( input ) {
+    var additionalFields = $( input ).attr('data-additional-fields');
+
+    if ( $( input ).is(':checked') ) {
+      $( additionalFields ).show();
+    } else {
+      $( additionalFields ).hide();
+    }
+  }
+
+  // Show or hide additional fields when any radio button in fieldset changes
+  $( '.js-additional-field' ).on('change', function() {
+    var showField = $( this ).find( '.js-additional-field_trigger');
+    toggleAdditionalFields( showField );
+  } );
+
+
+  // Show or hide additional fields on init
+  $( '.js-additional-field' ).each( function ( i ) {
```

Comment:
Good point. It should be usable without JS, but to make it truly usable we need some kind of messaging that would be hidden to js users to explain that the additional fields only need to be filled out in their specific situations. We could probably add that to the next sprint so we can get this in today.